### PR TITLE
Update to include CMPowerLogViewer

### PR DESCRIPTION
**Pull request by neilwurtz**
CMPowerLogViewer can cause confusion when marketing as OneTrace.  Including this in the documentation.

### Summarize the change in the pull request title

Describe your change, specifically *why* you think it's needed.

Fixes #Issue_Number (if necessary)

### CODE REVIEW

**Comment by PRMerger9** [comment]
@neilwurtz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@neilwurtz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@neilwurtz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @neilwurtz for the contribution. I'm discussing with the engineering team, stay tuned...

File: sccm/core/support/support-center-onetrace.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ OneTrace works with many types of log files, such as:
 
 OneTrace installs with Support Center. Find the Support Center installer on the site server at the following path: `cd.latest\SMSSETUP\Tools\SupportCenter\SupportCenterInstaller.msi`.
 
+OneTrace application can be found <InstallLocation>\CMPowerLogViewer.exe
```
Comment (aczechowski) [inline sccm/core/support/support-center-onetrace.md]: Can you reword this a bit, and add the default path. For example:

```suggestion
By default, the OneTrace application is installed at `"C:\Program Files (x86)\Configuration Manager Support Center\CMPowerLogViewer.exe`.
```

**Comment by PRMerger20** [comment]
@neilwurtz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by aczechowski** [comment]
#sign-off 

*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 9 October) around 11:00 Pacific time. No further action needed from you.

**Comment by aczechowski** [comment]
#sign-off

**Comment by aczechowski** [comment]
closing/reopening to see if automation will kick in...

**Comment by aczechowski** [comment]
#sign-off

**Comment by aczechowski** [comment]
FYI, opened a support case to investigate the automation issue.

**Pull request merged**